tests/providers/apache/druid/hooks/test_druid.py

```diff
@@ -26,7 +26,8 @@
 from airflow.providers.apache.druid.hooks.druid import DruidDbApiHook, DruidHook, IngestionType
 
 
-class TestDruidHook:
+@pytest.mark.db_test
```

Comment:
Part of the test was DB, part was not. I decided to split the class to avoid having multiple decorators.